metadata.rb

```diff
@@ -1,35 +1,27 @@
-# Copyright 2017 Google Inc.
+# Copyright:: 2021 Google LLC
+#
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name 'google-glogging'
+name 'google_chef_agents'
```

Comment:
We should sync on this, as Chef recommended practice is to have the cookbook name, namespace, and repo name all the same. However, Chef has deprecated having hyphens (like the repo name) in Cookbook names and namespaces. This will be disallowed in new releases. Please see this question and answer:
https://discourse.chef.io/t/what-is-the-right-style-to-name-a-cookbook/7532

Reply:
Yeah, let's sync about this in Hangout chat or the meeting tomorrow.